CMake/FindRocksDB.cmake

```diff
@@ -73,6 +73,8 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 elseif(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
   # On Linux
   set(ROCKSDB_LIBRARY_NAMES ${ROCKSDB_LIBRARY_NAMES} librocksdb.so)
+elseif(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
+  set(ROCKSDB_LIBRARY_NAMES ${ROCKSDB_LIBRARY_NAMES} librocksdb.so)
```

Comment:
We don't need these variables. These find modules are mostly drop ins from the respective projects. In the second-level CMake we're only pulling in the libs&switches in the static variable: https://github.com/facebook/osquery/blob/master/osquery/CMakeLists.txt#L15